database_apis/users_database.py

```diff
@@ -117,12 +119,44 @@ def logout(self):
         
     def get_current_user(self):
         return self.user
+    
+    def add_auto_login_information(self, mac_address : str, email: str):
```

Comment:
This function adds the auto login information of the user computer's mac address and the date and time the user enables the auto login function in the login page to the database users table.